agent/agent.go

```diff
@@ -887,8 +887,10 @@ func (a *Agent) makeServer() (*consul.Server, error) {
 	if err != nil {
 		return nil, err
 	}
-	if err := a.setupKeyrings(config); err != nil {
-		return nil, fmt.Errorf("Failed to configure keyring: %v", err)
+	if !a.config.DisableKeyringFile {
```

Comment:
I'm ok leaving it disable too, since it defaults to enabled and we have a lot of other things like that in the config. I'll leave it up to you @kyhavlov - either way works.